pxi.py

```diff
@@ -170,107 +180,104 @@ def parse_xml(self, xml_str):
+                    elif child.tag == "TTL":
+                        # self.ttl.load_xml(child)
+                        # self.ttl.init()
+                        pass
+                        
+                    elif child.tag == "DAQmxDO":
+                        # self.daqmxdo.load_xml(child)
+                        # self.daqmxdo.init()
                         pass
-                elif child.tag == "TTL":
-                    # self.ttl.load_xml(child)
-                    # self.ttl.init()
-                    pass
-                elif child.tag == "DAQmxDO":
-                    # self.daqmxdo.load_xml(child)
-                    # self.daqmxdo.init() # called setup in labview
-                    pass
-                elif child.tag == "timeout":
-                    try:
-                        # get timeout in [ms]
-                        self.measurement_timeout = 1000*float(child.text)
-                    except ValueError as e:
-                        self.logger.error(f"{e} \n {child.text} is not valid "+
-                                          f"text for node {child.tag}")
-
-                elif child.tag == "cycleContinuously":
-                    cycle = False
-                    if child.text.lower() == "True":
-                        cycle = True
-                    self.cycle_continuously = cycle
-
-                elif child.tag == "camera":
-                    pass
-                    # set up the Hamamatsu camera
-                    try:
+
+                    elif child.tag == "timeout":
+                        try:
+                            # get timeout in [ms]
+                            self.measurement_timeout = 1000*float(child.text)
+                        except ValueError as e:
+                            msg = f"{e} \n {child.text} is not valid "+
+                                              f"text for node {child.tag}"
+                            raise XMLError(self, msg)
+
+                    elif child.tag == "cycleContinuously":
+                        cycle = False
+                        if child.text.lower() == "True":
+                            cycle = True
+                        self.cycle_continuously = cycle
+
+                    elif child.tag == "camera":
+                        # set up the Hamamatsu camera
                         self.hamamatsu.load_xml(child)  # Raises ValueError
                         self.hamamatsu.init()  # Raises IMAQErrors
-                    # Errors below are raised due to improper settings, should not stop executions
-                    except (IMAQError, AssertionError, ValueError, KeyError) as e:
-                        self.handle_errors(e, "initializing hamamatsu")
+
+                    elif child.tag == "AnalogOutput":
+                        # set up the analog_output
+                        # self.analog_output.load_xml(child)
+                        # self.analog_output.init() # setup in labview
```

Comment:
Uncomment methods that have been implemented so the remaining work to be done is clear.

Reply:
I had done this, then remembered you had commented out anything that would inhibit a successful dry run of the server. I will change it back.